ClosedXML/Excel/CalcEngine/Functions/Statistical.cs

```diff
@@ -117,23 +118,18 @@ static object CountA(List<Expression> p)
         static object CountBlank(List<Expression> p)
         {
             var cnt = 0.0;
-            foreach (Expression e in p)
+
+            if (p == null || p.Count != 1 ||
```

Comment:
This test is already catered for. At the top of the class, this statement implies that only 1 parameter is allowed: `ce.RegisterFunction("COUNTBLANK", 1, CountBlank);`